### PR TITLE
Various proposed changed

### PR DESCRIPTION
**Pull request by jakirkham**
Includes various proposed changes from staged-recipes.
- Change `fn` for better caching/offline use.
- Adds `pkg-config` as a run time dependency.
- Adds `license_family`.
- Add myself as a maintainer.
- Bumps the build number to get the effect of all changes.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -22,6 +22,7 @@ requirements:
     - nose >=1.0
   run:
     - python
+    - pkg-config  # [unix]
```
Comment (jakirkham) [inline recipe/meta.yaml]: Would it be worth adding `m2w64-pkg-config` here, @mingwandroid?

**Review by pelson — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (mingwandroid) [reply recipe/meta.yaml]: I think that's fine, Python just calls the executable, right?

Comment (jakirkham) [reply recipe/meta.yaml]: Yep, that's all it's doing. Went ahead and added it.

**Comment by jakirkham** [comment]
Thoughts on this, @lbdreyer ?

**Comment by lbdreyer** [comment]
Looks good :)

Thanks @jakirkham

**Comment by jakirkham** [comment]
Thanks. 😄

**Pull request merged**